allensdk/brain_observatory/ecephys/stimulus_analysis/flashes.py

```diff
@@ -39,21 +39,20 @@ class Flashes(StimulusAnalysis):
 
     def __init__(self, ecephys_session, col_color='color', trial_duration=0.25, **kwargs):
         super(Flashes, self).__init__(ecephys_session, trial_duration=trial_duration, **kwargs)
-
         self._metrics = None
 
         self._colors = None
-        self._col_color = col_color # 'color'
+        self._col_color = col_color
 
         if self._params is not None:
             self._params = self._params.get('flashes', {})
             self._stimulus_key = self._params.get('stimulus_key', None)  # Overwrites parent value with argvars
         else:
             self._params = {}
 
-        # self._trial_duration = 0.25
-        self._module_name = 'Flashes'
-
+    @property
+    def name(self):
+        return 'Flashes'
```

Comment:
are these names supposed to line up exactly with stimulus names?

Reply:
No, these are just generic stimulus class names. The stimulus_key can be a bit more complex.